src/components/field-text-area/styled.ts

```diff
@@ -15,6 +15,18 @@ const Language = styled.span`
   text-transform: uppercase;
   text-align: center;
   font-weight: bold;
+
+  ${({ isReadOnly }) =>
+    isReadOnly &&
+    css`
+      top: -3px;
+      margin: 0;
+      padding: 3px 5px;
+      border-radius: 4px;
+      width: 34px;
+      text-transform: lowercase;
+      background: ${({ theme }) => theme.fdk.colors.neutrals.skyblue};
```

Comment:
Finnes det en fargepalett vi skal bruke, 'skyblue' har jeg ikke sett før i designsystemet?

Reply:
Jeg mener denne fargen kom direkte fra skissene og vi ikke hadde navn for den.